test/Libraries/PackageManagerTests/PackageLoaderTests.cs

```diff
@@ -237,6 +268,150 @@ public void LoadingConflictingCustomNodePackageDoesNotGetLoaded()
                                              x.FullName == "Test.EvenOdd") == 1);
         }
 
+        [Test]
+        public void LoadingConflictingCustomNodePackage_AfterPlacingNode_DoesNotGetLoaded()
+        {
+            var loader = GetPackageLoader();
+            var libraryLoader = new ExtensionLibraryLoader(CurrentDynamoModel);
+
+            loader.PackagesLoaded += libraryLoader.LoadPackages;
+            loader.RequestLoadNodeLibrary += libraryLoader.LoadNodeLibrary;
+
+          
+            var packageDirectory = Path.Combine(TestDirectory, "pkgs", "EvenOdd");
+            var packageDirectory2 = Path.Combine(TestDirectory, "pkgs", "EvenOdd2");
+            var package1 = Package.FromDirectory(packageDirectory,CurrentDynamoModel.Logger);
+            var package2 = Package.FromDirectory(packageDirectory2, CurrentDynamoModel.Logger);
+            loader.LoadPackages(new Package[] { package1,package2 });
+
+            // There are 2 packages loaded directly
+            Assert.AreEqual(2, loader.LocalPackages.Count());
+
+            var entries = CurrentDynamoModel.SearchModel.SearchEntries.OfType<CustomNodeSearchElement>();
+
+            // Check that conflicting custom node package "EvenOdd2" is not installed
+            Assert.IsTrue(entries.Count(x => Path.GetDirectoryName(x.Path).EndsWith(@"EvenOdd2\dyf")) == 0);
+            Assert.IsTrue(entries.Count(x => Path.GetDirectoryName(x.Path).EndsWith(@"EvenOdd\dyf") &&
+                                             x.FullName == "Test.EvenOdd") == 1);
+
+            var customNodeInstance = CurrentDynamoModel.CustomNodeManager.CreateCustomNodeInstance(Guid.Parse("3f19c484-d3f3-49ba-88c2-6c386a41f6ac"));
+            //this will reset the info.
+            this.CurrentDynamoModel.CurrentWorkspace.AddAndRegisterNode(customNodeInstance);
+
+            //load again.
+            loader.LoadPackages(new Package[] { package1, package2 });
```

Comment:
Why are you loading the packages again here? One of them is already loaded so it's not going to load again, is it? The following assertions are going to work even without this `LoadPackages` call again, won't they?

Reply:
I want to try loading package2 again - after the customNodeInstance is placed - because this action resets the customNodeInfo to not include the package info - so I want to verify that loading package2 (which was not loaded the first time) - is still not loaded even after creating an instance of this custom node.

Reply:
I could be more explicit and not try to load package1 again for clarity.